src/libsync/websocket.h

```diff
@@ -0,0 +1,45 @@
+#ifndef WEBSOCKET_H
+#define WEBSOCKET_H
+
+#include <QWebSocket>
+
+class TestWebSocket;
+
+namespace OCC {
+
+class AbstractWebSocket : public QObject
```

Comment:
Still unsure I see the point of this.

Reply:
@er-vin Maybe it was meant to be used with a stub in unit tests? Fake WebSocket would inherit the abstract one?